etl-pipeline/processes/grin/download.py

```diff
@@ -52,7 +52,7 @@ def download_barcode(self, barcode):
             decrypted_ocr_package = self.decrypt_ocr_package(
                 barcode, tmp_ocr_package, tmp_dir
             )
-            self.upload_unpacked_ocr_files(barcode, ocr_dir, decrypted_ocr_package)
+            self.unpack_and_upload_mets_file(barcode, ocr_dir, decrypted_ocr_package)
```

Comment:
Does this need a merge with main? I think these are changes from last week right?